src/components/Common/PageOverlay.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+	import Spawnable from './Spawnable.svelte';
+
+	export let target = null;
+	let node = null;
+	let update = false;
+
+	window.addEventListener('resize', () => onResize());
+
+	function refresh() {
+		const targetRect = target.getBoundingClientRect();
+		node.style.transform = 'translate(' + targetRect.left + 'px, ' + (window.scrollY + targetRect.top) + 'px)';
```

Comment:
It may break the layout if located close to the bottom of the page: ![dropdownatbottom](https://user-images.githubusercontent.com/8793817/232210315-49c9bd17-d4e2-4f80-902e-91e6dc31d296.gif)